coreMacros/src/main/scala-3/me/shadaj/scalapy/py/Facades.scala

```diff
@@ -68,6 +68,8 @@ object FacadeImpl {
 
   def creator[T <: Any](using Type[T], Quotes): Expr[FacadeCreator[T]] = 
     import quotes.reflect.*
+    if(TypeRepr.of[T].typeSymbol.flags.is(Flags.Trait))
+      report.error(s"${TypeRepr.of[T].show} should be a class, not a trait!")
```

Comment:
Can we adjust this a bit to clarify that this is only a rule for Scala 3? Perhaps something like "Facade definitions in Scala 3 must be classes, but T was defined as a trait"?

Reply:
Yes sure :) thank you for the tip!